tailwind.config.js

```diff
@@ -0,0 +1,23 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./src/**/*.{js,jsx,ts,tsx}"],
+  theme: {
+    extend: {
+      fontFamily: {
+        'inter' : ['Inter', 'sans-serif'],
```

Comment:
our font family is poppins for the main website could you please change the font to reflect that?

Reply:
Ah okay, I thought it was Inter because the new design for the website is in Inter.

Reply:
@Kevin-Tan25 Just added the fonts